src/components/pages/jobs/Jobs.tsx

```diff
@@ -0,0 +1,327 @@
+        </Link>
+      </div>
+    </div>
+  );
+};
+
+const FilterJobs: React.FC = () => {
+  const [forBedrifter, setForBedrifter] = useState(false);
+
+  const parseTags = (data: JobItem[]) => {
+    const tags: string[] = [];
+    data?.forEach((job) => {
+      job.tags.forEach((tag) => {
+        if (!tags.includes(tag)) {
+          tags.push(tag);
+        }
+      });
+    });
+    return tags;
+  };
+  const ctxHook = useContext(FilterContextHook);
+  const [uniqueTags, setUniqueTags] = useState<string[]>(
+    parseTags(ctxHook.allJobs)
+  );
+  const [hoverSort, setHoverSort] = useState(false);
+  useEffect(() => {
+    setUniqueTags(parseTags(ctxHook.allJobs));
+  }, [ctxHook?.context, ctxHook.allJobs]);
+
+  return (
+    <div className={styles.filterWrapper}>
+      <TextField
+        label={'Search'}
+        type={'Search'}
+        onChange={(e) => {
+          ctxHook?.setContext({
+            ...ctxHook?.context,
+            search_string: e.target.value,
+          });
+        }}></TextField>
+      <div
+        className={styles.filterSortWrapper}
+        onMouseEnter={() => {
+          setHoverSort(true);
+        }}
+        onMouseLeave={() => {
+          setHoverSort(false);
+        }}
+        onClick={(e) => {
+          ctxHook?.setContext({
+            ...ctxHook.context,
+            sort_date: !ctxHook.context.sort_date,
+          });
+        }}>
+        <small>Sort by date</small>
+        <Icon
+          color={hoverSort ? '#f09667' : '#444658'}
+          type={
+            ctxHook?.context.sort_date
+              ? 'sort-amount-up-alt'
+              : 'sort-amount-down-alt'
+          }></Icon>
+      </div>
+      <hr />
+      <div className={styles.filterTagWrapper}>
+        {uniqueTags.map((tag, key) => {
+          return (
+            <div key={key}>
+              <Chip label={tag} active={false} />
+            </div>
+          );
+        })}
+      </div>
+      <div style={{ display: 'flex', marginTop: '2rem' }}>
+        <div
+          className={styles.companyButton}
+          onClick={() => {
+            setForBedrifter(!forBedrifter);
+          }}>
+          For bedrifter!
+        </div>
+      </div>
+      {forBedrifter && (
+        <div className={styles.fade_in}>
+          <p
+            style={{
+              fontSize: '0.8rem',
+            }}>
+            Har din organisasjon -eller bedrift lyst til å dele ledige
+            stillinger, sommerjobber eller liknende hos oss i Tromsøstudentenes
+            Dataforening? Send en epost til
+            <a
+              href="mailto:leder@td-uit.no"
+              style={{
+                textDecoration: 'None',
+                color: 'rgba(240, 150, 103, 0.5)',
+              }}>
+              {' '}
+              leder@td-uit.no{' '}
+            </a>
+            for å få mer informasjon om dette!
+          </p>
+        </div>
+      )}
+    </div>
+  );
+};
+
+const Jobs: React.FC = () => {
```

Comment:
Maybe JobPage is a more suitable name and the jobItem is imported from molecules